airgun/views/common.py

```diff
@@ -440,7 +440,8 @@ def search(self, query):
         if not self.is_searchable():
             return None
         self.searchbox.search(query)
-        self.title.click()
+        if hasattr(self, 'title'):
+            self.title.click()
```

Comment:
if you see this behaviour in many search widgets? if not I think it better to move and fix in a particular view only. This could fail other tests as this is a common location.